generators/Exercise.cs

```diff
@@ -69,10 +69,19 @@ protected virtual string RenderTestMethodBody(CanonicalDataCase canonicalDataCas
         protected virtual TestMethodBody CreateTestMethodBody(CanonicalDataCase canonicalDataCase)
         {
             if (canonicalDataCase.ExceptionThrown != null)
+            {
                 return new TestMethodBodyWithExceptionCheck(canonicalDataCase, CanonicalData);
+            }
 
             if (canonicalDataCase.Expected is bool)
+            {
                 return new TestMethodBodyWithBooleanCheck(canonicalDataCase, CanonicalData);
+            }
+
+            if(canonicalDataCase.Expected is null)
```

Comment:
Oh man, that would bother me too. I didn't even notice that. Usually Visual Studio just automatically tosses that in for me.